python/cac_tripplanner/templates/home.html

```diff
@@ -85,7 +87,8 @@ <h1>Places we love</h1>
             </header>
             <ul class="place-list">
                 {% for destination in destinations %}
-                <li class="place-card no-origin" data-destination-id="{{ destination.id }}"
+                <li class="place-card {% if destination.start_date %}event-card {% endif %} no-origin"
```

Comment:
Good idea to have in the JSON properties set by `set_destination_properties`, but in the Django template here it's using objects from context instead of the API JSON. I'll see about making it a model property on the base class (which will also add it to the JSON).